apps/test-bundles/webpackUtils.js

```diff
@@ -73,6 +73,59 @@ function createWebpackConfig(entries) {
 // Files which should not be considered top-level entries.
 const TopLevelEntryFileExclusions = ['index.js', 'version.js', 'index.bundle.js'];
 
+function createFluentNorthstarFixtures() {
+  const packageName = '@fluentui/react-northstar';
+  const distPath = path.dirname(require.resolve(packageName).replace('commonjs', 'es'));
+  const packagePath = path.resolve(distPath, 'components');
+  fs.readdirSync(packagePath).forEach(itemName => {
+    const isFolder = fs.statSync(path.join(packagePath, itemName)).isDirectory();
+
+    if (isFolder && itemName) {
+      const importStatement = `import { ${itemName} } from '${packageName}'; console.log(${itemName})`;
+      try {
+        const folderName = getFolderName(packageName);
+        const entryPath = path.join('fixtures/', folderName, `${itemName}.js`);
+        fs.outputFileSync(entryPath, importStatement, 'utf-8');
+      } catch (err) {
+        console.log(err);
+      }
+    }
+  });
+}
+
+function createFluentFixtures() {
```

Comment:
This name might be a bit confusing because elsewhere "fluent" refers to northstar/v0 (which itself is weird, but while the other convention exists, it's best not to introduce an additional meaning). Maybe either `createFluentReactFixtures` or `createFluentVersion8Fixtures`?